src/Hl7.Fhir.Specification/Specification/Terminology/ExternalTerminologyService.cs

```diff
@@ -27,18 +27,32 @@ public ExternalTerminologyService(BaseFhirClient client)
 
         public async Task<Parameters> ValueSetValidateCode(Parameters parameters, string id = null, bool useGet = false)
         {
-            if (string.IsNullOrEmpty(id))
-                return await Endpoint.TypeOperationAsync<ValueSet>(RestOperation.VALIDATE_CODE, parameters, useGet).ConfigureAwait(false) as Parameters;
-            else
-                return await Endpoint.InstanceOperationAsync(constructUri<ValueSet>(id), RestOperation.VALIDATE_CODE, parameters, useGet).ConfigureAwait(false) as Parameters;
+            if (parameters.TryGetDuplicates(out var duplicates))
+            {
+                throw Error.Argument($"List of input parameters contains the following duplicates: {string.Join(", ", duplicates)}");
+            }
+            {
```

Comment:
I think you wanted here an `else`, but  an `else clause` is not needed. The `if clause` throws already an exception.